activerecord/test/cases/fixtures_test.rb

```diff
@@ -310,6 +310,10 @@ def test_has_many_through_with_renamed_table
     assert_equal load_has_and_belongs_to_many['parrots_treasures'], rows['parrot_treasures']
   end
 
+  def test_has_and_belongs_to_many_order
+    assert_equal ['parrots', 'parrots_treasures'], load_has_and_belongs_to_many.keys
```

Comment:
I do agree but testing the actual load seems quite hard. I can test that `insert_fixture` was called for each table but MiniTest provides no way to test the order of these calls, unlike RSpec, which has an `ordered` method.